model/map-jpa/src/main/java/org/keycloak/models/map/storage/jpa/updater/liquibase/MapJpaLiquibaseUpdaterProvider.java

```diff
@@ -163,7 +164,7 @@ protected Status validateChangeSet(Liquibase liquibase, String changelog) throws
     }
 
     private Liquibase getLiquibase(Class modelType, Connection connection, String defaultSchema) throws LiquibaseException {
-        LiquibaseConnectionProvider liquibaseProvider = session.getProvider(LiquibaseConnectionProvider.class);
+        LiquibaseConnectionProvider liquibaseProvider = session.getProvider(LiquibaseConnectionProvider.class, MapLiquibaseConnectionProviderFactory.PROVIDER_ID);
```

Comment:
This should be a separate SPI with a separate provider that should not specify the `MapLiquibaseConnectionProviderFactory.PROVIDER_ID` as a specific provider ID, rather rely on the one picked by the system. 
```suggestion
        MapLiquibaseProvider liquibaseProvider = session.getProvider(MapLiquibaseProvider.class);
```